PyDSS/cli/run.py

```diff
@@ -46,8 +46,15 @@
     show_default=True,
     help="Enable verbose log output."
 )
+@click.option(
+    "--dry-run",
+    is_flag=True,
+    default=False,
+    show_default=True,
+    help="Dry run for getting estimated space."
```

Comment:
We need inform the user that this is the size before compression.  Optional for this PR: document some typical compression rates.  I'm not sure how typical `tests/data/project` will be.  I'm guessing something around 90%.

Reply:
Does it necessary to run a bunch of tests to calculate the `true` space required (not the current estimated one)? then calculate this compression ratio. Or you think it's enough with this rough number, 90%.